doc/ImuFactor.lyx

```diff
@@ -227,16 +227,62 @@ preintegrated_
 
 \begin_layout Standard
 The main function of a factor is to calculate an error.
- The easiest case to look at is the NavState variant in ImuFactor2, which
- is given as:
+ This is done exactly the same in both variants: 
 \begin_inset Formula 
 \begin{equation}
-\Delta X_{ij}=X_{j}-\hat{X_{ij}}\label{eq:imu-factor-error}
+e(X_{i},X_{j})=X_{j}\ominus\widehat{X_{j}}\label{eq:imu-factor-error}
 \end{equation}
 
 \end_inset
 
+where the predicted NavState 
+\begin_inset Formula $\widehat{X_{j}}$
+\end_inset
+
+ at time 
+\begin_inset Formula $t_{j}$
+\end_inset
+
+ is a function of the NavState 
+\begin_inset Formula $X_{i}$
+\end_inset
+
+ at time 
+\begin_inset Formula $t_{i}$
+\end_inset
+
+ and the preintegrated measurements 
+\begin_inset Formula $PIM$
+\end_inset
+
+:
+\begin_inset Formula 
+\[
+\widehat{X_{j}}=f(X_{i},PIM)
+\]
+
+\end_inset
+
+The noise model associated with this factor is assumed to be zero-mean Gaussian
+ with a 
+\begin_inset Formula $9\times9$
```

Comment:
Ok, though Frank wanted this to be specific since we have different sections for ImuFactor and CombinedFactor. I'll add a corresponding paragraph in the `Combined IMU Factor` subsection.

Reply:
thanks!